TCAT/Utilities/Analytics.swift

```diff
@@ -218,3 +208,13 @@ struct WhatsNewCardDismissedPayload: Payload {
 
     let actionDescription: String
 }
+
+/// Log get errors
+struct GetErrorPayload: Payload {
+    static let eventName: String = "Get Error"
+    let deviceInfo = DeviceInfo()
+    
```

Comment:
Final nit i promise but we can remove this newline

Reply:
all the other payloads had a line after `deviceInfo` so i thought i would keep it consistent, or should i remove all the lines for the other payloads?

Reply:
Hmm we can j keep then no problem